{"event": "verbose", "uuid": "ba3ae426-9cb8-4e83-ac40-218bca035616", "counter": 7, "stdout": "\u001b[0;31mThe error appears to be in '/runner/project/playbooks/output.yml': line 17, column 6, but may\u001b[0m", "start_line": 6, "end_line": 7, "runner_ident": "ece86fb9-416c-4eca-aae2-a9b6d34412dd", "created": "2021-11-13T14:20:57.585938"}